recipes/libiec61850/all/conanfile.py

```diff
@@ -0,0 +1,68 @@
+import os
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import get, copy
+
+required_conan_version = ">=2.1"
+
+class Libiec61850Conan(ConanFile):
+    name = "libiec61850"
+    description = "An open-source library for the IEC 61850 protocols."
+    license = "LGPL-3.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://libiec61850.com/libiec61850"
+    topics = ("iec61850", "mms", "goose", "sampled values")
+
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    languages = ["C"]
+    implements = ["auto_shared_fpic"]
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["BUILD_EXAMPLES"] = False
+        tc.variables["BUILD_TESTS"] = False
```

Comment:
rule of thumb: if the expectation for a user is to pass them via `-D` in the command line (as is this case) these should always be `cache_variables` - this is also covered in our docs:
https://docs.conan.io/2/reference/tools/cmake/cmaketoolchain.html#variables

`tc.variables` should be reserved for things CMake expects in a toolchain file.